docs/user_guides/mlops/serving/deployment.md

```diff
@@ -192,6 +193,10 @@ Inference batcher are deployment component that apply batching to the incoming i
 
 Resources include the number of replicas for the deployment as well as the resources (i.e., memory, CPU, GPU) to be allocated per replica. To learn about the different combinations available, see the [Resources Guide](resources.md).
 
+## API protocol
+
+Hopsworks supports both REST and gRPC as the API protocols to send inference requests to model deployments. To learn about REST and gRPC API protocols for model deployments, see the [API Protocol Guide](api-protocol.md).
```

Comment:
In general, you use gRPC when you need lower latency inference requests. To learn more about the REST and gRPC API protocols